airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -785,7 +785,7 @@
 - name: Jira
   sourceDefinitionId: 68e63de2-bb83-4c7e-93fa-a8a9051e3993
   dockerRepository: airbyte/source-jira
-  dockerImageTag: 0.3.1
+  dockerImageTag: 0.3.2
```

Comment:
revert this change, it will be bumped automatically by publishing scripts

Reply:
Done :heavy_check_mark: